File: src/assets/usage-examples-code/timers/has_timer__named/has_timer_named-1-basic-usage.cs
using System;
using SplashKitSDK;

class Program
{
    static void Main()
    {
        SplashKitSDK.Timer myTimer = SplashKit.CreateTimer("MyTimer");

        // Check if SplashKit has a timer named "MyTimer"
        bool hasMyTimer = SplashKit.HasTimer("MyTimer");

        Console.WriteLine($"SplashKit have a timer named 'MyTimer'? {hasMyTimer}");

        SplashKit.FreeAllTimers();
    }
}

File: src/assets/usage-examples-code/timers/has_timer__named/has_timer_named-1-basic-usage.cpp
#include "splashkit.h"
#include <iostream>

int main()
{
    timer myTimer = create_timer("MyTimer");

    // Check if SplashKit has a timer named "MyTimer"
    bool hasMyTimer = has_timer("MyTimer");

    std::cout << "SplashKit have a timer named 'MyTimer'? " << std::boolalpha << hasMyTimer << std::endl;
    free_all_timers();

    return 0;
}
